phase0/progress_report.md

```diff
@@ -5,27 +5,27 @@ The project works by taking User Input regarding courses they want, and filters
 
 # CRC Model Summary
 * Controller:
-  * CommandLineInterface, maps user input to calling commands
+  * controllers.CommandLineInterface, maps user input to calling commands
```

Comment:
I think u don't need to do `controllers.CommandLineInterface` here. Same for the rest of progress_report.md. Basically u can just refer to classes by their class name, not by their full name including package. Because it's a reasonable assumption that reader can find the Class you are referring to by class name only

Reply:
Oh wait this is from the refactoring tool right? Ok just go thru all the text files and revert that b/c i think the refactoring tool changed the text to reflect the package

Reply:
i reverted those, what do you think about the design document description for the packages?